tbx/project_styleguide/templates/patterns/molecules/title-filters/title-filters.html

```diff
@@ -0,0 +1,32 @@
+{% load wagtailcore_tags %}
+
+<div class="title-filters">
+    <div class="title-filters__title">
+        <h1 class="heading heading--one">{{ item.title }}</h1>
+    </div>
+    {# todo: see if this is needed when implementing the team page #}
+    {% comment %} {% if meta %}
+        {{ meta }}
+    {% endif %} {% endcomment %}
+    <div class="title-filters__tags">
+        {% if tags %}
```

Comment:
Should this wrap `<div class="title-filters__tags">` so there won't be an empty div if there are no tags?

Reply:
good point. I'm not sure we'd use this component if there weren't filters, but good to get the logic correct. Fixed in c0f05047f3b8c32aa16c44c36352644681786abe.